wgsl/index.bs

```diff
@@ -1879,6 +1879,10 @@ variable_storage_decoration
 
 </pre>
 
+The `const` identifiers are immutable. When a `const` identifier is declared without the corresponding type,
```

Comment:
Identifiers aren't immutable.  It's the value that the identifier denotes that is immutible.

Reply:
Pipeline-overrideable const's might not have an initializer.

Like discussed in the meeting for 'var',  if the const has no initializer, then the type must be provided.

Reply:
This section describes the statements, where the grammar requires an initializer:
```
CONST (IDENT | variable_ident_decl) EQUAL short_circuit_or_expression
```